src/modules/rangeproof/tests_impl.h

```diff
@@ -547,8 +547,95 @@ static void test_rangeproof(void) {
     }
 }
 
+static void test_single_value_proof(uint64_t val) {
+    unsigned char proof[5000];
+    secp256k1_pedersen_commitment commit;
+    unsigned char blind[32];
+    unsigned char blind_out[32];
+    unsigned char nonce[32];
+    const unsigned char message[1] = " "; /* no message will fit into a single-value proof */
+    unsigned char message_out[sizeof(proof)] = { 0 };
```

Comment:
I'm going to leave this be -- I did consider using a nonzero value, but decided against it, because
* 0 is very easy to notice in debug output (though I suppose so would any constant value)
* the failure mode in message decoding will be to output random-looking data (since rewinding extracts from a RNG and then xors the output). So it's hard to see how rewinding could fail in a way that would be confused with constant 0s